### PR TITLE
feat: new block style

### PR DESCRIPTION
**Pull request by sebbalex**
This pr aims to have a different look and feel

### CODE REVIEW

**Comment by libremente** [comment]
Nice @sebbalex, however the users PNGs are not loaded.

File: src/App.jsx [hunk 1]
```diff
@@ -82,7 +82,9 @@ function App() {
         />
       </div>
       <h4 className="text-center mt-5">Vaccinazione per fasce di età</h4>
```
Comment (libremente) [inline src/App.jsx:84]:
```suggestion
      <h4 className="text-center mt-5">Vaccinazioni per fasce di età</h4>
```

**Review by libremente — CHANGES_REQUESTED** [review]

**Comment by sebbalex** [comment]
> Nice @sebbalex, however the users PNGs are not loaded.

yep, fixed, thanks

**Review by libremente — APPROVED** [review]

**Review by bfabio — APPROVED** [review]
:rocket:

**Pull request merged**